js/src/components/google-mc-account-card/connected-card.js

```diff
@@ -18,8 +18,15 @@ import useDispatchCoreNotices from '.~/hooks/useDispatchCoreNotices';
 import useApiFetchCallback from '.~/hooks/useApiFetchCallback';
 import { useAppDispatch } from '.~/data';
 
-const ConnectedCard = ( props ) => {
-	const { googleMCAccount } = props;
+/**
+ * Renders a Google Merchant Center account card UI with connected account information.
+ * It also provides a switch button that lets user connect with another account.
+ *
+ * @param {Object} props React props.
+ * @param {{ id: number }} props.googleMCAccount A data payload object contains user's Google Merchant Center account ID.
```

Comment:
```suggestion
 * @param {{ id: number }} props.googleMCAccount A data payload object containing the user's Google Merchant Center account ID.
```

Reply:
- props.googleMCAccount - A data payload object contains user's Google Merchant Center account ID. 

First is referring to googleMCAccount , but then Is saying That a Data Payload object, in general, contains Google Merchant Center Account ID. 

Could be like "props.fishEater - A person eats fish" 

- props.googleMCAccount -  A data payload object containing the user's Google Merchant Center account ID.

It is clear that we are actually referring to googleMCAccount being the payload object containing Google Merchant Center account ID. Also "A data payload that contains the user's Google Merchant Center account ID." works fine. 

Could be like "props.fishEater - A person that eats fish",  "props.fishEater - A person eating fish"  

It was definitely a super minor correction and I understood perfectly the intent of the property. But I as a non-native English speaker also love when someone points out my very usual English mistakes. Definitely I'm not the best to talk about English 🙈

Reply:
Thanks for the explanation! I fixed them in https://github.com/woocommerce/google-listings-and-ads/pull/1374/commits/cb541963947610af17db28c699ad8799b15fd383.